docs/docs/vrf/overview.md

```diff
@@ -110,7 +112,27 @@ const call = await account.execute([
 
 **Ensure that you call `consume_random` with the same `Source` as used in `request_random`.**
 
-By following these steps, you can integrate the VRF Provider into your Starknet contract and generate verifiable random numbers for your onchain game or application.
+### Important: Adding VRF to Policies
+
+When using the Cartridge Controller with VRF, make sure to add the VRF contract address and the `request_random` method to your policies. This allows the controller to pre-approve VRF-related transactions, ensuring a seamless experience for your users.
```

Comment:
Hm I think we could add this as a default policy to controller since it is a noop